orc8r/cloud/go/syncstore/store_writer.go

```diff
@@ -196,9 +196,7 @@ func (l *syncStore) collectGarbageSQL(tracked []string) error {
 			return nil, err
 		}
 		_, err := sqorc.ExecInTx(l.db, nil, nil, txFn)
-		if err != nil {
-			errs = multierror.Append(errs, errors.Wrapf(err, "collect garbage for table %+v", tableName))
-		}
+		errs = multierror.Append(errs, errors.Wrapf(err, "collect garbage for table %+v", tableName))
```

Comment:
Removing this nil check might make things more complicated if we intend to get rid of errors.Wrapf later? We will have to reintroduce it then, I suppose.

Reply:
Yes, you are right. Since it is merged now we'll have to change it in the PR that gets rid of Wrapf (#12682); I will arrange that.